src/display/api.js

```diff
@@ -1742,7 +1742,8 @@ var WorkerTransport = (function WorkerTransportClosure() {
             var size = width * height;
             var rgbaLength = size * 4;
             var buf = new Uint8Array(size * components);
-            var tmpCanvas = createScratchCanvas(width, height);
+            var canvasFactory = new DOMCanvasFactory();
```

Comment:
[Here](http://logs.glob.uno/?c=mozilla%23pdfjs&s=27+Jan+2017&e=27+Jan+2017#c56478) is the conversation with @yurydelendik, when i am trying to solve this issue. I think i got confused of passing `canvasFactory` via `CanvasGraphics`. I will fix it. Thanks for clearing.